docs/FeatureManagement/Samples/NoFilters.sample.json

```
@@ -0,0 +1,60 @@
+{
+    "feature_management": {
+        "feature_flags": [
+            {
+                "id": "BooleanTrueFeatureFlag",
+                "description": "A simple feature flag, that always returns true.",
+                "enabled": true,
+                "conditions": {
+                    "client_filters": []
+                }
+            },
+            {
+                "id": "BooleanFalseFeatureFlag",
+                "description": "A simple feature flag, that always returns false.",
+                "enabled": false,
+                "conditions": {
+                    "client_filters": []
+                }
+            },
+            {
+                "id": "StringTrueFeatureFlag",
+                "description": "A simple feature flag, that always returns false.",
+                "enabled": "true",
+                "conditions": {
+                    "client_filters": []
+                }
+            },
+            {
+                "id": "StringFalseFeatureFlag",
+                "description": "A simple feature flag, that always returns true.",
+                "enabled": "false",
+                "conditions": {
+                    "client_filters": []
+                }
+            },
+            {
+                "id": "InvalidEnabledFeatureFlag",
+                "description": "A simple feature flag, that has an invalid enabled field.",
+                "enabled": "invalid",
+                "conditions": {
+                    "client_filters": []
+                }
+            },
+            {
+                "id": "MinimalFeatureFlag",
+                "enabled": true
+            },
+            {
+                "id": "NoEnabled"
+            },
+            {
+                "id": "NoConditions",
```

```suggestion
                "id": "EmptyConditions",
```

BTW, I think this has been covered in MinimalFeatureFlag.

---

I don't think so, it validates that the client_filters field is optional.